docs/charts/polar.md

```diff
@@ -96,6 +96,7 @@ These are the customisation options specific to Polar Area charts. These options
 | `startAngle` | `number` | `-0.5 * Math.PI` | Starting angle to draw arcs for the first item in a dataset.
 | `animation.animateRotate` | `boolean` | `true` | If true, the chart will animate in with a rotation animation. This property is in the `options.animation` object.
 | `animation.animateScale` | `boolean` | `true` | If true, will animate scaling the chart from the center outwards.
+| `angles` | `array` | `(2*Math.PI)/serieCount` | Computed angles to draw each item arc.
```

Comment:
I don't think `serie` is a word. It also doesn't match the usual terminology. We have datasets which contain data points

Reply:
Oups sorry, you're right, this is actualy the French word. 

I suggest to put `[]` as default value and explain the behavior more in details in the description column with the following : 

- If this option is not set, `polar circumference = 2 * PI` and each arc will be `(2 * PI) / (visible datasets count)`.
- Otherwise, `polar circumference = sum(options.angles)`.
  - If `options.angles.length < (visible datasets count)`
    - Undefined angles will be set to `(2 * PI) / (visible datasets count)`.
    - Defined angles will be recomputed to fill proportionaly the remaining polar circumference (`(polar circumference) - sum(undefined angles)`).
  - If `options.angles.length > (visite datasets count)`
    - Angles will be recomputed proportionaly to fill the entire polar circumference.

What do you think ?